zkevm-circuits/src/state_circuit.rs

```diff
@@ -529,6 +515,14 @@ fn queries<F: Field>(meta: &mut VirtualCells<'_, F>, c: &StateCircuitConfig<F>)
         + meta.query_advice(first_different_limb.bits[4], Rotation::cur());
     let mpt_update_table_expressions = c.mpt_table.table_exprs(meta);
 
+    let meta_query_word =
+        |metap: &mut VirtualCells<'_, F>, word_column: word::Word<Column<Advice>>, at: Rotation| {
+            word::Word::new([
+                metap.query_advice(word_column.lo(), at),
+                metap.query_advice(word_column.hi(), at),
+            ])
+        };
```

Comment:
I was thinking about what is a good general API for querying the word limbs. In the keccak PR, I did

`hash_word.map(|col| meta.query_advice(col, Rotation::cur())),`

But it feels verbose.